Sources/Algorithms/Stride.swift

```diff
@@ -226,9 +226,15 @@ extension StrideCollection: Collection {
     offsetBy n: Int,
     limitedBy limit: Index
   ) -> Index? {
-    let distance = i == endIndex
-      ? -((base.count - 1) % stride + 1) + (n - 1) * -stride
-      : n * -stride
+    // We typically use the ternary operator but this significantly increases compile
+    // times when using Swift 5.3.2
```

Comment:
Small formatting nit:

```suggestion
    // We typically use the ternary operator but this significantly increases
    // compile times when using Swift 5.3.2
```

Reply:
Thanks @xwu, I've merged it with a note to squash it into the other commit.